src/main/kotlin/com/exactpro/th2/common/utils/event/EventBatcher.kt

```diff
@@ -32,7 +32,8 @@ import kotlin.concurrent.withLock
  * for a group has elapsed or number of events in it has reached `maxBatchSize`.
  */
 class EventBatcher(
-    private val maxBatchSize: Int = 100,
+    private val maxBatchSizeInBytes: Long = 1_024 * 1_024,
+    private val maxBatchSizeInItems: Int = 100,
```

Comment:
Do we need to batch by size in items and in bytes at the same time? 
If not then instead maybe pass a function which will calculate new batch size and test it against `maxBatchSize`?